trame_server/controller.py

```diff
@@ -29,10 +30,21 @@ class Controller:
     >>> ctrl.on_data_change.clear(set_only=True) # add, remove, discard, clear
     """
 
-    def __init__(self, trigger_decorator, trigger_name):
+    def __init__(self, server):
+        super().__setattr__("_server", server)
         super().__setattr__("_func_dict", {})
-        self.trigger = trigger_decorator
-        self.trigger_name = trigger_name
+
+    @property
+    def trigger(self):
+        return self._server.trigger
```

Comment:
Those are functions. Are they going to work when exposed as properties?

Reply:
Yes they do.